### DISCUSSION TITLE
Missing folders and submodules

### DISCUSSION THREAD

**Discussion by edy14431** [micropython/micropython#12242] [Ideas]
I would like to suggest including the missing folders and submodules in the third-party software tree present in the LICENSE file. micropython-lib could be included in that tree for example. There are other folders and submodules missing.

**Comment by sosi-deadeye** [comment]
I guess the file is written manually.
Maybe you can automate this with: https://github.com/topics/license-scan
The disadvantage is the required time to parse all files.
